lms/templates/student_account/account_settings.underscore

```diff
@@ -2,16 +2,16 @@
 <div class="account-settings-container">
     <div class="wrapper-header">
         <h2 class="header-title"><%- gettext("Account Settings") %></h2>
-        <ul class="left list-inline account-nav">
+        <div class="left list-inline account-nav" role="tablist">
             <% _.each(accountSettingsTabs, function(tab) { %>
-            <li class="account-nav-item">
-                <button id="<%- tab.id %>" data-name="<%- tab.name %>" <% if (tab.class) { %> aria-describedby="header-subtitle-<%- tab.name %>" <% } %> class="account-nav-link <%- tab.class %>" ><%- tab.label %></button>
-            </li>
+                <button id="<%- tab.id %>" aria-controls="<%- tab.name %>-tabpanel" <% if (tab.class == 'active') { %> tabindex="0" aria-selected="true" aria-expanded="true" <% } else { %> tabindex="-1" aria-selected="false" aria-expanded="false" <% } %> data-name="<%- tab.name %>" aria-describedby="header-subtitle-<%- tab.name %>" class="tab account-nav-link <%- tab.class %>" role="tab"><%- tab.label %></button>
```

Comment:
NIT:  Can we put the `tab.label` and the `</button>` each on their own lines just to make this slightly easier to parse.  There is a lot going on here